website/documentation/050-tutorials/content/howto/add-node-to-cluster/_index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Manually adding node to an existing cluster
+description: "Manually adding node to an existing cluster"
+type: tutorial-page
+level: advanced
+index: 10
+reviewer: Gaurav Gupta
+status: 
+last_reviewed: 
+category: Operation
+scope: app-developer
+---
+
+## Term clarification
+Here we will look at steps on how to add a node to an existing cluster without the support of Gardener. Such a node will not be managed by Gardener, and if it goes down for any reason, Gardener will not be responsible to replace it.
```

Comment:
This could be useful where a customer needs specifically configured nodes, which are not yet supported by Gardener. For example: an end-user who wants some workload that require runnc instead of runc as container runtime.
@vasu1124 could shed some more light on this if required.

Reply:
ok, understood. I think this consideration should be be mentioned in the content of this tutorial otherwise those reading it may wonder why this is useful

Reply:
This is essentially a self help for users, who like the control plane to be delivered as a service by Gardener, but want to test options with the kubelet and its plugins, runtimeclass, CRI etc. We cant elaborate or pre-envision all cases. 
Configureable Custom OS overlap, but even then, the Gardener service owner might want to restrict the options to cover a defined SLA.